packages/os/cfsignal.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Send signal to CloudFormation Stack
+After=network-online.target settings-applier.service
+Requires=settings-applier.service
+Wants=network-online.target
```

Comment:
Why doesn't the unit declare a stronger dependency on `network-online.target`?

Reply:
Since there's no need to. `Wants=` + `After=` is sufficient to express "start this service after network is up". This is the same across for all our other services.